modules/streaming/src/main/scala/io/circe/streaming/package.scala

```diff
@@ -6,23 +6,58 @@ import io.circe.jawn.CirceSupportParser
 import io.iteratee.{ Enumeratee, Enumerator }
 
 package object streaming {
-  final def stringParser[F[_]](implicit F: ApplicativeError[F, Throwable]): Enumeratee[F, String, Json] =
-    new ParsingEnumeratee[F, String] {
-      protected[this] final def parseWith(p: AsyncParser[Json])(in: String): Either[ParseException, Seq[Json]] =
-        p.absorb(in)(CirceSupportParser.facade)
-    }
 
-  final def byteParser[F[_]](implicit F: ApplicativeError[F, Throwable]): Enumeratee[F, Array[Byte], Json] =
-    new ParsingEnumeratee[F, Array[Byte]] {
-      protected[this] final def parseWith(p: AsyncParser[Json])(in: Array[Byte]): Either[ParseException, Seq[Json]] =
-        p.absorb(in)(CirceSupportParser.facade)
-    }
+  /**
+    * String parser of JSON values wrapped in a single array
+    */
+  final def stringArrayParser[F[_]](implicit F: ApplicativeError[F, Throwable]): Enumeratee[F, String, Json] =
+    stringParser(AsyncParser.UnwrapArray)
+
+  /**
+    * String parser of JSON values delimited by whitespace/newline
+    */
+  final def stringStreamParser[F[_]](implicit F: ApplicativeError[F, Throwable]): Enumeratee[F, String, Json] =
```

Comment:
I like it. Even though I can totally see how stream parser vs. array parser may be misinterpreted as the "parser that takes stream or array as an input", I think these names our best bet (given we document the difference).